test/performance/README.md

```diff
@@ -1,128 +1,189 @@
+
+   ```bash
+   $ export WORK_DIR=$HOME/perf-test/clusterloader2
+   ```
+
+2. Configure clusterloader2 to generate KS performance workloads:
+
+   Starting from a local directory containing this KubeStellar git repo, run the following script based on your cluster environment:
+
+   a) If using plain K8s clusters:
+
+   ```bash
+   $ ./setup-clusterloader2.sh
+   ```
+
+   In plain Kubernetes environments, a modified version of the kube-burner [cluster-density](https://github.com/kube-burner/kube-burner/tree/main/examples/workloads/cluster-density) workload will be generated per namespace. This workload consists of the following objects:
+
+   - 1 deployments, with two pod replicas (pause), mounting 2 secrets, 2 config maps
+   - 3 services, the first service points to the TCP/8080 port of the deployments, respectively.
+   - 10 secrets containing a 2048-character random string.
+   - 10 config maps
+
+   b) If using OpenShift clusters: 
+
+   ```bash
+   $ ./setup-clusterloader2.sh --env ocp
+   ```
+
+    In Openshift environments, a modified version of the kube-burner [cluster-density-ms](https://github.com/kube-burner/kube-burner-ocp/tree/478bb42e1842a94ca3210d26a08633b70a443005/cmd/config/cluster-density-ms) workload will be generated per namespace. This workload consists of the following objects:
+
+    - 1 image stream.
+    - 4 deployments, each with two pod replicas (pause), mounting 4 secrets, 4 config maps, and
+      1 downward API volume each.
+    - 2 services, each pointing to the TCP/8080 and TCP/8443 ports of the first and second
+      deployments, respectively.
+    - 1 edge route pointing to the first service.
+    - 20 secrets containing a 2048-character random string.
+    - 10 config maps containing a 2048-character random string
+
+
+3. Configure the parameters of your workload:  
 
    a) cd into the load configuration directory
 
-```bash
-$ cd  perf-tests/clusterloader2/testing/load
-```
-   b) create the `stress-test` namespace - you workload needs to be deployed into this exact namespace (temporary solution)
-
-```bash
-$ kubectl apply -f - <<EOF
-apiVersion: v1
-kind: Namespace
-metadata:
-  name: stress-test
-  labels:
-    app.kubernetes.io/cluster: wec1
-EOF
-```
-
-   c) configure your sample workload: 
-   
-```bash
-$ vi test-workload.yaml
-```
+   ```bash
+   $ cd  $WORK_DIR
+   ```
   
-   d) configure the parameters to create the workload traffic (e.g., RandomizedLoad, SteppedLoad, Sequence, etc.)
+   b) configure the parameters to create the workload traffic (e.g., RandomizedLoad, SteppedLoad, Sequence, etc.)
    
-```bash
-$ vi performance-test-config.yaml
-``` 
+   ```bash
+   $ vi performance-test-config.yaml
+   ``` 
+
+   More specifically, configure the following parameters: 
+
+   - namespaces: number of namespaces to be created in your cluster to deploy the worklod defined in step-2 above (*default value: 1*)
```

Comment:
Why would a user ever want a value other than 1?

Reply:
To generate higher workload in their performance tests. That's one of the experiments that I am planning to conduct when experiment variable is the "number of bindingPolicies".